### ISSUE TITLE
 Toggle Theme Feature

### ISSUE THREAD

**Issue by Vishalk91-4** [GrabBits/GrabBits_Website#90]
**Is your feature request related to a problem? Please describe.**
Enabling  Dark Mode / Light Mode Feature in the entire Website
 
**Describe the solution you'd like**
Dark Mode View
![Screenshot 2023-05-27 154252](https://github.com/GrabBits/GrabBits_Website/assets/116670999/dd7c25b7-cb72-45f3-82ab-28b250870234)

Current / Light Mode View
![Screenshot 2023-05-27 154418](https://github.com/GrabBits/GrabBits_Website/assets/116670999/cf6ca650-7f0d-4d30-9e64-f17e1ca3aa13)

**Describe alternatives you've considered**
Elaborate on any alternative solutions or features that you have already evaluated as potential approaches to address the problem at hand. Include a succinct evaluation of their respective advantages and disadvantages in comparison to your desired solution.

**Additional context**
Include any supplementary information, contextual details, relevant screenshots, or supporting materials that can aid in better understanding and supporting your feature request.

**Comment by Spyware007** [comment]
@Vishalk91-4 Assigned to you!
Ping me on discord if you need any help!

**Comment by Vishalk91-4** [comment]
@Spyware007 I've been working on the footer issue also, so I'll be submitting the PR to it by Sunday is it fine, I'm having my college end sems also
Hope you understand

**Comment by Spyware007** [comment]
@Vishalk91-4 Are you done implementing this?

**Comment by Vishalk91-4** [comment]
Yes some changes are done, it's taken time because I am having end sems

**Comment by Spyware007** [comment]
@Vishalk91-4 
Can you please give me updates regarding this issue?

**Comment by devmusiczz** [comment]
@Spyware007 i can do it more faster. Please assign this issue to me under gssoc. I will complete this in 1 or max 2 days.

**Comment by Vishalk91-4** [comment]
@Spyware007 , I have made changes but as said yesterday not completed and I have my remaining exams are from 19-22, so I won't be doing much in between
I think you could assign both of us and which one you like you can merge that

**Comment by Spyware007** [comment]
@Vishalk91-4  Fine
@devmusiczz You can start working on this asap! 🚀
All the best 🙌💯

**Comment by devmusiczz** [comment]
can you please reassign. i mistakenly unassigned it.

**Comment by Grab-Bits** [comment]
@Vishalk91-4 You can take your time and complete it as per your exams schedule!
This issue is only assigned to you now!
